mmpretrain/models/backbones/eva02.py

```diff
@@ -0,0 +1,691 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Sequence
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import build_activation_layer
+from mmcv.cnn.bricks.drop import build_dropout
+from mmcv.cnn.bricks.transformer import PatchEmbed
+from mmengine.model import BaseModule, ModuleList
+from mmengine.model.weight_init import trunc_normal_
+
+from mmpretrain.registry import MODELS
+from ..utils import build_norm_layer, resize_pos_embed, to_2tuple
+
+
+class RotaryEmbedding(BaseModule):
```

Comment:
this module could be mv into `mmpretain/models/utils/postion_encoding.py`. unit test is needed.